build.gradle

```diff
@@ -28,36 +22,47 @@ plugins {
 
 apply plugin: 'io.spring.nohttp'
 
-//def gitPresent = new File('.git').exists()
+def gitPresent = new File('.git').exists()
 
-//if(gitPresent) {
-//	apply plugin: 'org.ajoberstar.grgit'
-//}
+if(gitPresent) {
+	apply plugin: 'org.ajoberstar.grgit'
+}
 
 description = 'Spring for Apache Pulsar'
 
 ext {
-//	if (gitPresent) {
-//		modifiedFiles =
-//			files(grgit.status().unstaged.modified).filter{ f -> f.name.endsWith('.java') || f.name.endsWith('.kt') }
-//	}
+	linkHomepage = 'https://github.com/spring-projects-experimental/spring-pulsar'
+	linkIssue = 'https://github.com/spring-projects-experimental/spring-pulsar/issues'
+	linkScmUrl = 'https://github.com/spring-projects-experimental/spring-pulsar'
+	linkScmConnection = 'https://github.com/spring-projects-experimental/spring-pulsar.git'
+	linkScmDevConnection = 'git@github.com:spring-projects-experimental/spring-pulsar.git'
+	docResourcesVersion = '0.2.5'
+
+	javadocLinks = [
+			'https://docs.oracle.com/en/java/javase/17/docs/api/',
+			'https://docs.spring.io/spring-framework/docs/current/javadoc-api/'
+	] as String[]
+
+	if (gitPresent) {
+		modifiedFiles =
+				files(grgit.status().unstaged.modified).filter{ f -> f.name.endsWith('.java') || f.name.endsWith('.kt') }
+	}
 
-	assertjVersion = '3.21.0'
-	awaitilityVersion = '4.1.1'
+	assertjVersion = '3.22.0'
+	awaitilityVersion = '4.2.0'
 	googleJsr305Version = '3.0.2'
 	hamcrestVersion = '2.2'
-	hibernateValidationVersion = '6.2.3.Final'
-	jacksonBomVersion = '2.13.2.20220328'
+	hibernateValidationVersion = '7.0.4.Final'
+	jacksonBomVersion = '2.13.3'
 	jaywayJsonPathVersion = '2.6.0'
 	junit4Version = '4.13.2'
 	junitJupiterVersion = '5.8.2'
```

Comment:
I think we can instead pull in the junit bom under dependency mgmt which would get rid of some of these version specified I believe. Same as what [spring-framework](https://github.com/spring-projects/spring-framework/blob/73d92d66b938a5ff9080442396bed13c421d1f8f/build.gradle#L309) does.

Reply:
Ok, i just followed what Spring-Kafka does. We can do that. Feel free to make that change once this PR is merged.